gwpy/frequencyseries/frequencyseries.py

```diff
@@ -435,6 +435,10 @@ def to_pycbc(self, copy=True):
             epoch = None
         else:
             epoch = self.epoch.gps
+        if self.f0.to('Hz').value > 0.0:
+            raise ValueError("Cannot convert FrequencySeries to PyCBC with "
+                             "f0 = {0}. Starting frequency must be equal "
+                             "to 0 Hz.".format(self.f0))
```

Comment:
It might be cleaner here to use f-strings:

```suggestion
            raise ValueError(
                "Cannot convert FrequencySeries to PyCBC with f0 = {self.f0}. "
                "Starting frequency must be equal to 0 Hz."
            )
```